neo/SmartContract/TriggerType.cs

```diff
@@ -3,6 +3,8 @@
     public enum TriggerType : byte
     {
```

Comment:
Would it be better for this to be an enum style bit mask?  

That way if an existing contract checks for `Verification`, even if the contract is `VerificationR` it will match.

Reply:
Hmm, since `Verification` is typically just a `CheckWitness` of the contract owner, and this is a receive rather than a send, existing contracts will just fail too with `false`.